ui/src/tasks/actions/index.ts

```diff
@@ -483,3 +487,20 @@ export const getLogs = (taskID: string, runID: string) => async (
     dispatch(setLogs([]))
   }
 }
+
+export const convertToTemplate = (taskID: string) => async (
+  dispatch
+): Promise<void> => {
+  try {
+    dispatch(setExportTemplate(RemoteDataState.Loading))
+
+    const task = await client.tasks.get(taskID)
+    const taskTemplate = taskToTemplate(task)
```

Comment:
does this filter out the task token label for an org?

Reply:
no it doesn't. thanks!